packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -111,9 +118,34 @@ class BottomSheet extends StatefulWidget {
   /// If true, the bottom sheet can be dragged up and down and dismissed by
   /// swiping downwards.
   ///
+  /// This applies to the content below the drag handle, if showDragHandle is true.
+  ///
   /// Default is true.
   final bool enableDrag;
 
+  /// Specifies whether a drag handle is shown.
+  ///
+  /// The drag handle appears at the top of the bottom sheet. The default color is
+  /// [ColorScheme.onSurfaceVariant] with an opacity of 0.4 and can be customized
+  /// using dragHandleColor. The default size is Size(32,4) and can be customized
+  /// with dragHandleSize.
+  ///
+  /// Defaults to [BottomSheetThemeData.showDragHandle]. If that is also null,
+  /// defaults to true when [ThemeData.useMaterial3] is true, and otherwise
+  /// defaults to false.
+  final bool? showDragHandle;
+
+  /// The bottom sheet drag handle's color.
+  ///
+  /// Defaults to [BottomSheetThemeData.dragHandleColor].
+  /// If that is also null, defaults to [ColorScheme.onSurfaceVariant]
+  /// with an opacity of 0.4.
+  final Color? dragHandleColor;
```

Comment:
If the theme's dragHandleColor is `MaterialStateProperty<Color?>?` then this should be too (and we should specify what states it's resolved against).